pytest_fixtures/component/maintain.py

```diff
@@ -8,6 +8,17 @@
 from robottelo.constants import SATELLITE_MAINTAIN_YML
 from robottelo.hosts import Satellite
 
+synced_repos = pytest.StashKey[dict]
+
+
+@pytest.fixture(scope='module')
+def module_stash(request):
+    """Module scoped stash for storing data between tests"""
+    # Please refer the documentation for more details on stash
+    # https://docs.pytest.org/en/latest/reference/reference.html#stash
+    request.node.stash[synced_repos] = {}
```

Comment:
I would rather use just bool instead of dict. There's no need to store repos within stash function.

Reply:
We still need to return the repos at the end. When everything gets reran, it has no access to the repos unless we did some searching through the API. This way we just save them in a dictionary and return them when they've already been created.